CMakeLists.txt

```diff
@@ -41,8 +41,10 @@ endif()
 if (CMAKE_C_COMPILER_ID STREQUAL "Intel")
   if (MSVC)
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /fp:precise")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Qstd=c++11")
   else ()
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fp-model precise")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I wonder if we shouldn't set a CMake 3.1 minimum and do this sometime soon (not for this patch, which looks fine IMO)
```
set (CMAKE_CXX_STANDARD 11)
```

Reply:
I looked into this briefly, and it seems that this was brought in after CMake 2.6.0 (the current minimum version at the top of the file)